src/edgepi/adc/adc_state.py

```diff
@@ -65,9 +79,9 @@ def __query_state(self, adc_property: ADCProperties) -> PropertyValue:
         adc_property_value = adc_property.value.values[adc_property_bits]
         _logger.debug(
             (
-                f"query_state: query_property='{adc_property}',"
-                " adc_property_bits={hex(adc_property_bits)},"
-                f" adc_property_value='{adc_property_value}'"
+                "query_state: query_property='{}',".format(adc_property) +
```

Comment:
format strings `f"{...}"` are eager, and so slow things down more. I opted to switch to lazy formatting where possible